src/fetch/payouts.py

```diff
@@ -72,7 +72,7 @@ def __init__(  # pylint: disable=too-many-arguments
         payment_eth: int,
```

Comment:
In line 46, you still have `reward_cow` as part of your numerical column list. I assume that is how it is stored in the DB. Is there a reason for differing from that naming?

Reply:
I assume that is the equivalent to `total_reward_cow`? If so then forgot what I said.

Reply:
The database does not know about primary or secondary rewards (since they are computed on the exterior). I don't think it's necessary to change that part, but we could if we want to.